contracts/0.8.25/vaults/StakingVault.sol

```diff
@@ -554,6 +614,39 @@ contract StakingVault is IStakingVault, OwnableUpgradeable {
      */
     event BeaconChainDepositsResumed();
 
+    /**
+     * @notice Emitted when ether is deposited to `DepositContract`.
+     * @param _sender Address that initiated the deposit.
+     * @param _deposits Number of validator deposits made.
+     * @param _totalAmount Total amount of ether deposited.
+     */
+    event DepositedToBeaconChain(address indexed _sender, uint256 _deposits, uint256 _totalAmount);
+
+    /**
+     * @notice Emitted when vault owner requests node operator to exit validators from the beacon chain
+     * @param _sender Address that requested the exit
+     * @param _pubkey Public key of the validator to exit
+     * @dev    Signals to node operators that they should exit this validator from the beacon chain
+     */
+    event ValidatorExitRequested(address _sender, string indexed _pubkey);
+
+    /**
+     * @notice Emitted when validator withdrawals are requested via EIP-7002
+     * @param _sender Address that requested the withdrawals
+     * @param _pubkeys Concatenated public keys of the validators to withdraw
+     * @param _amounts Amounts of ether to withdraw per validator
+     * @param _refundRecipient Address to receive any excess withdrawal fee
+     * @param _excess Amount of excess fee refunded to recipient
+     */
+    event ValidatorWithdrawalRequested(address indexed _sender, bytes _pubkeys, uint64[] _amounts, address _refundRecipient, uint256 _excess);
```

Comment:
```suggestion
    event ValidatorWithdrawalsTriggered(address indexed _sender, bytes _pubkeys, uint64[] _amounts, address _refundRecipient, uint256 _excess);
```

Reply:
do we have system-contract events?

Reply:
https://eips.ethereum.org/EIPS/eip-7002 does not specify any events

Reply:
Naming fixed in c6640ba7